test/e2e/max_pods.go

```diff
@@ -100,6 +100,12 @@ var _ = Describe("MaxPods", func() {
 		By(fmt.Sprintf("Starting additional %v Pods to fully saturate the cluster and trying to start a new one", podsNeededForSaturation))
 		expectNoError(RunRC(config))
 
+		// Make sure that cluster is saturated
```

Comment:
The comment says "make sure" but really you just log the number. Would it be better to compare len(pods.Items) and totalPodCapacity and fail the test if they're != ? Or at least only print a log message if they are != ?